website/content/en/docs/reference/configuration/sources/http_server.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: HTTP Server
-description: Receive logs emitted by an [HTTP](https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Client_request) server.
+description: Receives observability data from an [HTTP client request](https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Client_request)
```

Comment:
This is something that could go in the style guide I guess. but I notice this one reads different. The other to are just the verb, but this one leads in the possessive tense "Receives". It seems like that could be "Receive" instead, to match "Pull" and "Deliver".

Reply:
Y u no suggest so I can easily fix 😆